docs/Users_Guide/glossary.rst

```diff
@@ -5424,3 +5454,99 @@ METplus Configuration Glossary
 
    PREPBUFR_DATA_DIR
      .. warning:: **DEPRECATED:** Please use :term:`PB2NC_INPUT_DIR` instead.
+
+   GEN_VX_MASK_INPUT_DIR
+     Directory containing input data to GenVxMask. This variable is optional because you can specify a
+     full path to the input files using :term:`GEN_VX_MASK_INPUT_TEMPLATE`.
+
+     | *Used by:* GenVxMask
+     | *Family:* [dir]
+     | *Default:* None
+
+   GEN_VX_MASK_INPUT_TEMPLATE
+     Filename template of the input grid used by GenVxMask. This can be an input filename or a grid definition.
```

Comment:
This can be an input filename, named grid, or grid specification string (per NCAR/MET#1292).

Reply:
"grid definition" to me is what I understood as a string that defines a grid, i.e. G003 or MY_GRID, as opposed to specifying a full file path. The difference between a "named grid" or "grid specification string" is not clear to me based on the name. According to the MET GitHub issue, "G003" is a named grid, so I assume "grid specification string" is a user-defined string to define some region, but the two terms sound like they can be interchangeable.